cmake/civetweb-3rdparty-config.cmake

```diff
@@ -19,23 +19,18 @@ add_library(civetweb OBJECT
   ${_IMPORT_PREFIX}/src/md5.inl
 )
 
-target_compile_definitions(civetweb PRIVATE
-  CIVETWEB_API=
-  USE_IPV6
-  NDEBUG
-  NO_CGI
-  NO_CACHING
-  NO_SSL
-  NO_FILES
+target_compile_definitions(civetweb
+  PRIVATE
+    CIVETWEB_API=
+    USE_IPV6
+    NDEBUG
+    NO_CGI
+    NO_CACHING
+    NO_SSL
+    NO_FILES
 )
 
-target_include_directories(civetweb PUBLIC ${CIVETWEB_INCLUDE_DIRS})
-
-if(BUILD_SHARED_LIBS)
```

Comment:
I removed because the target is an OBJECT target which does not build any library.

Reply:
you need to keep those `if(BUILD_SHARED_LIBS)` blocks because they affect how the obect library object files are compiled.

Reply:
Oh, I see. Sorry. I reverted it.